erpnext/projects/doctype/timesheet/timesheet.py

```diff
@@ -7,7 +7,7 @@
 import frappe
 from frappe import _
 from frappe.model.document import Document
-from frappe.utils import flt, getdate, time_diff_in_hours
+from frappe.utils import flt, getdate, time_diff_in_hours, add_to_date
```

Comment:
```suggestion
from frappe.utils import add_to_date, flt, getdate, time_diff_in_hours
```

Use isort/precommit 😄